tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3541,3 +3542,63 @@ def combined_non_max_suppression(boxes,
     return gen_image_ops.combined_non_max_suppression(
         boxes, scores, max_output_size_per_class, max_total_size, iou_threshold,
         score_threshold, pad_per_class)
+  
+  
+@tf_export('image.median_filtering_2D')
+def median_filtering_2D(tf_img,filter_shapex=3,filter_shapey=3):
+    # This methods takes both 2D Tensor as well as 3D Tensor Images
```

Comment:
This comment should be a docstring so it's available in TF's generated documentation.